content/articles/domains-uk.markdown

```diff
@@ -65,6 +65,10 @@ To transfer a UK domain (i.e. example.uk or example.co.uk) into DNSimple, follow
 
 To transfer a UK domain out of DNSimple, [contact support](https://dnsimple.com/contact) and provide your domain name and your new registrar's IPS tag.
 
+## Updating the registrant name
+
+It is possible to change most values in the Whois record for a UK domain, but the registrant name must be changed through Nominet's Online Services (Nominet is the Registry for UK domains). It is possible to [register for Nominet Online Services](https://secure.nominet.org.uk/auth/login.html) with the email address you have in your registrant information and update the value for the registrant name with them.
```

Comment:
@jcaudle in some cases it's still possible for us to change it (e.g. it is, the first time as far as I remember or after a transfer).

I'd add something like

> but the registrant name in general should be changed

I'd also add to contact us if they have any question providing the UK domain name to check and the new value to use as a registrant.